src/custom-element.js

```diff
@@ -458,6 +461,25 @@ function createBaseCustomElementClass(win) {
       return this.implementation_.getLayoutPriority();
     }
 
+    /**
+     * Returns the promise that's resolved when the consent has been resolved.
+     * @return {!Promise}
+     */
+    waitForConsentPolicy_() {
+      const policyId = this.implementation_.getConsentPolicy();
```

Comment:
should we add this method to base element, and explicitly have it return `null` if not overridden?

Reply:
You mean move the `waitForConsentPolicy` method to base element instead of `getConsentPolicy`? What I don't want is to have individual base element class deal with `Services.consentPolicyServiceForDocOrNull(this.element)`?
Does that make sense?

Reply:
Nvm I realize it will be much easier for me to fix test by avoiding waiting for unused promise.resolve(). I'll rewrite the code a bit.